### PR TITLE
Remove @types/rollup from bundler

### PR DESCRIPTION
**Pull request by TimvdLippe**
These typings are now being shipped by Rollup itself: https://www.npmjs.com/package/@types/rollup

My `npm install` reordered some fields in the `package.json`. I assume this is new npm behavior :thinking: The content is still the same though. (For example `license` was duplicate specified)

### CODE REVIEW

File: packages/bundler/package.json [hunk 1]
```diff
@@ -3,8 +3,13 @@
   "version": "4.0.0-pre.7",
   "description": "Process Web Components into one output file",
   "homepage": "https://github.com/Polymer/tools/tree/master/packages/bundler",
-  "repository": "github:Polymer/tools",
-  "bugs": "https://github.com/Polymer/tools/issues",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Polymer/polymer-bundler.git"
```
Comment (usergenic) [inline packages/bundler/package.json]: This was actually changed to `"repository": "github:Polymer/tools"` on purpose.

File: packages/bundler/package.json [hunk 2]
```diff
@@ -3,8 +3,13 @@
...
+  },
+  "bugs": {
+    "url": "https://github.com/Polymer/polymer-bundler/issues"
+  },
```
Comment (usergenic) [inline packages/bundler/package.json]: Change back to `"bugs": "https://github.com/Polymer/tools/issues"`

**Comment by TimvdLippe** [comment]
@usergenic These changes were done automatically by npm 6.0. I think that if I revert them, the next time it will apply the changes again?

**Comment by usergenic** [comment]
I don't think so. I suspect this is symptomatic of moving to the monorepo from the separate bundler repo and residual something something in the matrix...  There's no reason to expect other than unaccounted-for-weirdness that this should happen.

**Comment by TimvdLippe** [comment]
Ugh, I hate the GitHub UI. Have to do this tomorrow on my laptop to clean up this mes....

**Comment by TimvdLippe** [comment]
Or actually, the last commite message is just bad, but the content is good. Apparently these repository and bugs fields were duplicate, so thats why my npm changed them 🤔

**Pull request merged**